bundles/org.openhab.binding.tasmotaplug/README.md

```diff
@@ -0,0 +1,77 @@
+# TasmotaPlug Binding
+
+This binding connects Tasmota flashed smart plugs with 1, 2, 3 or 4 relay channels to openHAB.
+The plug must report the status of the relay via the url `http://$PLUG_IP/cm?cmnd=Power` in order for the binding to work.
+See the [Tasmota Supported Devices Repository](https://templates.blakadder.com/plug.html) for a list of supported plugs.
+
+## Supported Things
+
+There is exactly one supported thing type, which represents any supported Tasmota smart plug.
+It has the `plug` id.
+Multiple Things can be added if more than one plug is to be controlled.
+
+## Discovery
+
+Discovery is not supported. All things must be added manually.
+
+## Thing Configuration
+
+At minimum, the host name must be specified.
+The refresh interval and number of channels can be overridden from the default.
+
+| Parameter   | Description                                                                             |
+|-------------|-----------------------------------------------------------------------------------------|
+| hostName    | The host name or IP address of the plug. Mandatory.                                     |
+| refresh     | Overrides the refresh interval of the plug status. Optional, the default is 30 seconds. |
+| numChannels | Number of channels on the Tasmota Plug (1-4). default 1                                 |
```

Comment:
Noticed a difference from the thing type config description (full stop with lower case letter after). Perhaps rephrase:
```suggestion
| numChannels | Number of channels on the Tasmota Plug (1-4). Optional, the default is 1                |
```